core/src/database/migrations.cpp

```diff
@@ -442,5 +442,18 @@ namespace ledger {
         template <> void rollback<7>(soci::session& sql) {
             sql << "DROP TABLE bech32_parameters";
         }
+
+        template <> void migrate<8>(soci::session& sql) {
+            sql << "CREATE TABLE ripple_memos("
```

Comment:
Could you please add a cascade delete here ?

Reply:
Sure I can!

Reply:
Done, on the `ripple_transactions` field, `ON DELETE CASCADE`.